makefile

```diff
@@ -0,0 +1,35 @@
+config_file=_config.yml
+devel_config_file=_config_devel.yml
+
+workdir=..
+deploy_dir=$(workdir)/deploy
+checkout_dir=$(workdir)/checkout
+
+# This target is invoked by a doc_independent job on the ROS buildfarm.
+html: build deploy
+
+# Clone a bunch of other repos part of the rosdistro and build the index.
+build:
+	mkdir -p $(deploy_dir)/cache
+	mkdir -p $(workdir)/cache/checkout
+	vcs import --input resources.repos $(workdir)
+	bundle exec jekyll build --verbose --trace --config=$(config_file)
+
+# deploy assumes download-previous and build were run already
+deploy:
+	# TODO(apojomovsky): re-enable deployment once the deploy URL is updated in
```

Comment:
FYI for reviewers: I just disabled the deployment step until the [deploy URL](https://github.com/ros2/rosindex/blob/clear_submodules/resources.repos#L16) gets updated, as pointed [here](https://github.com/ros2/rosindex/pull/8#discussion_r212490816).

Reply:
Solved in 216d428fea6373bc15e1bed8880156f364cf2254.